mecanum-driveXbox/robot.py

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env python3
+"""
+    This is a demo program showing how to use Mecanum control with the
+    MecanumDrive class.
+"""
+
+import wpilib
+from wpilib.drive import MecanumDrive
+#Import the Xbox specific control
+from wpilib.interfaces import GenericHID
+
+Hand = GenericHID.Hand
```

Comment:
Why did you import this if you're not going to use it?

Reply:
Yes, I will.

Reply:
I thought it was necessary for the non-enumerated version of the left and right sticks.